data/ecosystems/c/cardano.toml

```diff
@@ -20,6 +20,8 @@ github_organizations = [
   "https://github.com/CardanoSolutions",
   "https://github.com/input-output-hk",
   "https://github.com/stakhanovite-io",
+  "https://github.com/txpipe",
```

Comment:
I added TxPipe as its own subecosystem of Cardano in b221ea44f247b5433ebe5dfb966b79b725915f2d